Commands.md

```diff
@@ -1201,6 +1202,20 @@ or reset one file to certain commit
 $ git reset-file .htaccess dc82b19
 ```
 
+## git mr
+
+Checks out a merge request from GitLab. Usage: `git mr <ID|URL> [REMOTE=origin]`.
```

Comment:
IMHO, I suggest to use `[REMOTE]` directly instead of `[REMOTE=origin]`.
And it would be great if you could add more examples below.